run_kraken.py

```diff
@@ -193,6 +197,18 @@ def main(cfg):
             logging.error("Post scenarios are still failing at the end of all iterations")
             sys.exit(1)
 
+        # Fail if the application routes monitered by cerberus experience downtime during the chaos
+        if check_applicaton_routes and cerberus_enabled:
```

Comment:
thoughts on adding the call to this check into the get_status function so it gets called every time in the cerberus checks

Reply:
Nice idea, added the call to the cerberus get_status function. This will help with failing the run at the scenario level instead of the run level which can have multiple scenarios like you mentioned @paigerube14. PTAL.